deploy/examples/create-test-cluster.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#set -eEx
+KUBECTL="minikube kubectl --"
+ROOK_EXAMPLES_DIR="."
```

Comment:
well, in fact the script can be run from anywhere meanwhile you provide the examples dir. I'll change it to be a parameter and use "." by default in case nothing is provided.